keras/backend/jax/core.py

```diff
@@ -374,3 +373,11 @@ def device_scope(device_name):
     else:
         jax_device = device_name
     return jax.default_device(jax_device)
+
+
+def finfo(dtype):
+    return jnp.finfo(dtype)
```

Comment:
Since the values should be the same regardless of the backend, we don't really need to access the per-backend `finfo`, do we?

Reply:
You are right! I thought this op is useful in some attention-related ops like masking, etc. I tried testing them and they show the same values. We could simply make use of np.finfo.

Closing this PR.